engine/src/main/resources/assets/i18n/menu_en.lang

```diff
@@ -86,6 +86,7 @@
     "death-screen-title": "You are Dead",
     "default-binding": "Default binding",
     "delete-game": "Delete",
+    "details": "Details",
```

Comment:
Consider a more detailed (heh) key name here - there's actually already a `"world-config-preview": "Details...",`

While it might not be good to reuse the exact same key you could use a more explicit name like "save-game-details" even if the value is essentially the same.